typecheck: createUseOpenFetch not resolved from #imports

`nuxi typecheck` fails with:

```
.nuxt/open-fetch.ts(1,10): error TS2724: '"#imports"' has no exported member named 'createUseOpenFetch'. Did you mean 'createOpenFetch'?
```

The generated `.nuxt/open-fetch.ts` imports `createUseOpenFetch` from `#imports`, but `vue-tsc` can't resolve it — likely due to name collision with `createOpenFetch` (both registered as auto-imports from different runtime files).

`imports.d.ts` correctly re-exports `createUseOpenFetch`, but the type checker doesn't pick it up.

--- a/src/module.ts
+++ b/src/module.ts
@@ -115,6 +115,7 @@ export default defineNuxtModule<ModuleOptions>({
     nuxt.options.alias = {
       ...nuxt.options.alias,
       '#open-fetch': join(nuxt.options.buildDir, moduleName),
+      '#open-fetch-use-fetch': resolve('runtime/useFetch'),
       '#open-fetch-schemas/*': join(nuxt.options.buildDir, 'types', moduleName, 'schemas', '*'),
     }
 
@@ -218,7 +219,7 @@ export {}
       filename: `${moduleName}.ts`,
       getContents() {
         return `
-import { createUseOpenFetch } from '#imports'
+import { createUseOpenFetch } from '#open-fetch-use-fetch'
 ${schemas.map(({ name }) => `
 import type { paths as ${pascalCase(name)}Paths, operations as ${pascalCase(name)}Operations } from '#open-fetch-schemas/${kebabCase(name)}'
 `.trimStart()).join('').trimEnd()}

